### PR TITLE
Generalized Lower Level Service Testing Method 

### PR DESCRIPTION
**Pull request by t-fine**
After some experimentation in curling the `cpu_percent` and `gps` I realized that each can execute their primary function with a `/cpu` or `/gps` on the end of the curl url, which is just their service names after stripping of the `ibm.` so I do that and save it to a variable named `api` and add that to the curl ur.

### CODE REVIEW

File: edge/services/cpu_percent/Dockerfile.amd64 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM alpine:latest
-RUN apk --no-cache --update add gawk bc socat
+RUN apk --no-cache --update add gawk bc socat curl
```
Comment (bmpotter) [inline edge/services/cpu_percent/Dockerfile.amd64]: Why did you need to add curl to the cpu_percent and gps docker images? The new testing method uses curl on the host, so doesn't need it in the container, right? Or am i missing something?

Comment (t-fine) [reply edge/services/cpu_percent/Dockerfile.amd64]: After some experimentation it appears that they do require curl for this to work. I believe it's because I am exec-ing into the container and running curl, not curling from the outside since they don't "expose" their ip address, or whatever that docker flag is to expose their info.

Comment (bmpotter) [reply edge/services/cpu_percent/Dockerfile.amd64]: makes sense
